doc/README.md

```diff
@@ -0,0 +1,20 @@
+# Documentation
+
+## Installation and upgrading
+
+### Docker (not recommended for production)
+
+- [Check out Huginn with Docker](docker/install.md) Run a local Huginn installation using Docker
+
+### Manual installation
+
+Manual installation which will guide through the steps to install Huginn on any Ubuntu 12.04/14.04 or Debian 6/7 server.
```

Comment:
Manual installation *instructions* which will guide *you* through the steps to install Huginn on any Ubuntu 12.04/14.04 or Debian 6/7 server.